<launch>
	<node name="stage" pkg="stage_ros" type="stageros" args="$(find lab2)/world/playground.world " />
	<node name="c" pkg="lab2" type="checklines.py" output="screen" />
	<node name="splitMerge" pkg="lab2" type="splitMerge" output="screen"/>
	<node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find lab2)/lab2.rviz" />
</launch>
